### PR TITLE
improved start-hoodie addition

### LINKED ISSUE #103: Don’t set "hoodie-start" script if "start" script is set to "hoodie"
### 🤔 What you will need to know
Basic knowledge of Node.js and npm. 

### ❓ The Motivation
We improved the setup experience for new Hoodie apps by directly setting the "start" script in your app’s `package.json` file when you installed `hoodie` with `npm install --save hoodie` via https://github.com/hoodiehq/hoodie/pull/592

We also made sure that we don’t override an existing "start" script via https://github.com/hoodiehq/hoodie/pull/642, by setting "hoodie-start".

But if the "start" script is set to `"hoodie"`, then we don’t need to do anything. 

### 🐞 The Bug
If the `"start"` script is set to `"hoodie"`, `npm install --save hoodie` current sets the `"hoodie-start"` script. But it should not do anything if the start script is set to `"hoodie"` or something like `"hoodie --inMemory"`.

### :clipboard: Step by Step
- [x] 🙋 **Claim this issue**: Comment below (or assign yourself and continue at step 4 :)  
  Please 🙏 only claim if you want to start working on it within a day.  
  Once claimed we add you as contributor to this repository.
- [x] 👌 **Accept our invitation** to this repository. Once accepted, assign yourself to this repository
- [x] 👓 **Please review** our [Code of Conduct](http://hood.ie/code-of-conduct/)  
  In a nutshell: be patient and actively kind with us 😊
- [x] 🔄 **replace** the `up for grabs` label with `in progress`.
- [x] Unless you are a Hoodie Contributors team member,  fork the [hoodiehq/hoodie repository](https://github.com/hoodiehq/hoodie)
- [x] [**Setup the repository locally**](https://github.com/hoodiehq/hoodie/#testing). Ensure the tests pass.
- [x] We don’t currently have tests for the [`"postinstall"` hook](https://github.com/hoodiehq/hoodie/blob/87e5024c34093c60d843549408e5d650671701ea/package.json#L70), so don’t worry about it. Make the necessary change directly in [hoodie/bin/setup.js](https://github.com/hoodiehq/hoodie/blob/87e5024c34093c60d843549408e5d650671701ea/bin/setup.js)
- [x] 🔀 **Start a Pull Request**. Mention `closes hoodiehq/camp#103` in the description.  
  If this is your first, welcome :tada: :smile: [Here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.
- [x] 🏁 **Done** :+1: Replace the `in progress` label with `ready`. Ask in comments for a review :)

### 🤔❓ Questions

Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/)

### PR DESCRIPTION
**Pull request by gautamkrishnar**
fixes:https://github.com/hoodiehq/camp/issues/103

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
@gr2m :smile:

File: bin/setup.js [hunk 1]
```diff
@@ -18,9 +18,7 @@ if (saveRequested) {
 
   if (packageJson.scripts[ 'start' ]) {
     log.info('setup', 'start script already set to "' + packageJson.scripts[ 'start' ] +
-    ', you can start hoodie with "npm run start-hoodie" instead')
-
-    packageJson.scripts[ 'start-hoodie' ] = 'hoodie'
```
Comment (gr2m) [inline bin/setup.js:23]: sorry but removing `start-hoodie` entirely is not what we meant. We still want to keep it if the `start` script is set to something like `ember server` for example.

What we want is to add another `if () {}` statement in which we check if the current value of `packageJson.scripts[ 'start' ]` starts with `'hoodie'`. If it does, don’t do anything. If it does not, then set `packageJson.scripts[ 'start-hoodie' ] = 'hoodie'`. Does that make sense?

**Review by gr2m — CHANGES_REQUESTED** [review]

**Comment by gautamkrishnar** [comment]
Okay. I will make changes soon...

**Comment by gautamkrishnar** [comment]
@gr2m is just updated the code... Is this okay?

**Review by gr2m — APPROVED** [review]
great, thanks :)

**Comment by gr2m** [comment]
@gautamkrishnar the tests are failing, which keeps us from merging your pull request. You can run `npm test` to see them, they are linting errors. You can see them on Travis, too: https://travis-ci.org/hoodiehq/hoodie/builds/196663948#L657

**Comment by gautamkrishnar** [comment]
@gr2m fixed 🔪

**Comment by gr2m** [comment]
cheers :)

**Pull request merged**